### DISCUSSION TITLE
Mobile mode (sneak peak) [Jun 23]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5996] [Discord Threads]
> 🙋‍♂️ **lvarming**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Updated docs just now   
#### https://github.com/nocodb/nocodb/blob/develop/packages/noco-docs/content/en/engineering/development-setup.md#accessing-easter-egg-menu   
####    
#### You can toggle last option to enable beta features  
#  
> 🙋‍♂️ **lvarming**  
#### Thanks. I don't see any options to scan QR codes though..?  
#  
> 👨‍🦲 **dstala**  
####   
![image0](https://cdn.discordapp.com/attachments/1110519248058597429/1110870808861102080/Screenshot_2023-05-24_at_3.33.21_PM.png)  
#  
> 👨‍🦲 **dstala**  
#### you should be able to see this option after toggling beta feature in easter menu & enable `mobile mode` from project menu  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1110519248058597429)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
